pallets/xcmp-handler/Cargo.toml

```diff
@@ -13,6 +13,7 @@ targets = ['x86_64-unknown-linux-gnu']
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.0.0", features = ["derive"], default-features = false }
 scale-info = { version = "2.1", default-features = false, features = ["derive"] }
+log = { version = "0.4.17", default-features = false }
```

Comment:
did you mean to leave this here?

Reply:
I decided to add some logging back so this will still be there.